src/modules/logger/logger.service.ts

```diff
@@ -52,6 +52,10 @@ export class Logger extends NestLogger implements LoggerService {
     });
   }
 
+  setContext(ctx: string) {
+    this.context = ctx;
+  }
+
```

Comment:
so when using a custom logger we should set the context manually? or asking other way, I want to understand the purpose of this change?

Reply:
Haha, this is the same question I asked on your PR, I think: https://github.com/energywebfoundation/iam-cache-server/pull/146#discussion_r665036124
And the answer is the same. I guess `NestLogger` must have dropped their implementation of `setContext` in v8

Reply:
ah:D:D forgot about that:D yeah then its fine from my side<3